Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -40,13 +53,15 @@ protected virtual void Awake()
         {
             _stateManager.OnStateChanged += HandleStateChange;
             _inputSettings = new InputSettings();
+            SceneManager.LoadScene("WeaponManager", LoadSceneMode.Additive);
```

Comment:
I think the scene should be called something like `AdditiveWeaponManager` To make it clear that this scene should not be opened by itself! 

That was the convention I was trying to go for with the `AdditiveDebugScene`. What do you think?